provider/kubernetes/kubernetes.go

```diff
@@ -219,8 +219,14 @@ func (p *Provider) loadIngresses(k8sClient Client) (*types.Configuration, error)
 					}
 				}
 
-				if rule := getRuleForPath(pa, i); rule != "" {
-					templateObjects.Frontends[baseName].Routes[pa.Path] = types.Route{
+				rule, err := getRuleForPath(pa, i)
+				if err != nil {
+					log.Errorf("fail to get rule from ingress path: %s", err)
```

Comment:
My code coverage tool tells me that this line is not tested. Could you check please?